examples/ensemble/plot_feature_transformation.py

```diff
@@ -54,10 +53,9 @@
 rt = RandomTreesEmbedding(max_depth=3, n_estimators=n_estimator)
 rt_lm = LogisticRegression()
 rt.fit(X_train, y_train)
-rt_lm.fit(SelectFromModel(rt, prefit=True).transform(X_train_lr), y_train_lr)
+rt_lm.fit(rt.transform(X_train_lr), y_train_lr)
```

Comment:
This could probably be refactored to use a `rt_lm = make_pipeline(rt, lm)` to make the code more readable.